README.md

```diff
@@ -1,6 +1,6 @@
 # VSCode Color Info
 
-[Visual Studio Code](https://code.visualstudio.com) extension that provides quick information css colors.
+[Visual Studio Code](https://code.visualstudio.com) extension that provides quick information of colors in any programming language.
```

Comment:
Can you also please revert this change this it is not strictly related to the PR